### PR TITLE
Pbi/45295 datawarehouse new documentation

### PR DESCRIPTION
**Pull request by KabbaniReem**
Add entry for Datawarehouse documentation instead of power bi and make an overview page that links to all sub-chapters

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -282,11 +282,12 @@ nav:
...
   - Power Users:
+    - Datawarehouse: 
+         - Overview: power-users/datawarehouse/overview.md
+         - Prerequisites: power-users/datawarehouse/prerequisites.md
+         - Star Schema Model: power-users/datawarehouse/star-schema-model.md
+         - Power BI Example: power-users/datawarehouse/power-bi-example.md
+         - Excel Example: power-users/datawarehouse/excel-example.md
     - Microsoft Power Platform Integration:
       - Power Automate: 
-         - Connector Configuration and Usage: power-users/microsoft-platform-integration/power-automate/power-automate-symbio-connector-(coming-soon).md
```
Comment (thomashauser) [inline mkdocs.yml:287]: This line should not be deleted

File: docs/power-users/datawarehouse/overview.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+## Overview
+This chapter is dedicated to Datawarehouse service users such as Analysts to enable them build their own reports by means of Business Analysis tools such as Power BI and MS-Excel.
+The following sub-chapters are created for this aim:
+- [Prerequisites](https://github.com/ploetz-zeller/symbio-documentation/docs/power-users/datawarehouse/prerequisites.md) describe what you need to do in order to use the datawarehouse service
```
Comment (thomashauser) [inline docs/power-users/datawarehouse/overview.md:4]: Its possible the error is fixed, if you add blank lines between the paragraphs:
image

File: mkdocs.yml (hunk 1 above)
Comment (thomashauser) [reply mkdocs.yml:287]: Looks liked its fixed

**Review by thomashauser — CHANGES_REQUESTED** [review]

File: docs/power-users/datawarehouse/overview.md (hunk 2 above)
Comment (KabbaniReem) [reply docs/power-users/datawarehouse/overview.md:6]: fixed

**Review by thomashauser — APPROVED** [review]

**Pull request merged**